AccountingSystem/Client/ReportsClient.py

```diff
@@ -100,7 +100,7 @@ def getPlotToDirectory(self, plotName, dirDestination):
     transferClient = self.__getTransferClient()
     try:
       destFilename = "%s/%s" % (dirDestination, plotName)
-      destFile = file(destFilename, "wb")
+      destFile = open(destFilename, "wb")
```

Comment:
This currently returns without closing the file if an error happens:
```suggestion
      with open(destFilename, "wb") as destFile:
        retVal = transferClient.receiveFile(destFile, plotName)
```